techniques/system/server-roles/1.0/init-check.st

```diff
@@ -26,14 +26,8 @@ bundle agent generic_process_check_process(binary, initscript, name, force_resta
 
       "forced_trigger_${canoname}" expression => strcmp("${force_restart}", "true");
 
-      # Raise this class if we are handling a process that should be checked even on
-      # relay servers.
-      "process_exception_on_relay" expression => strcmp("${check_on_relay_server}", "true");
-
   processes:
 
-    root_server|process_exception_on_relay::
```

Comment:
I've removed all occurence of this condition within this bundle, because this bundle is now only ever called if it's necessary, so it should just do what it's told and not try and guess.